regex-automata/src/nfa/thompson/nfa.rs

```diff
@@ -1545,6 +1563,25 @@ pub enum State {
         /// satisfied.
         next: StateID,
     },
+    /// This is like a match state but for a lookaround expression
+    /// executing this state will write a `true` into the lookaround oracle at
+    /// index `look_idx`
+    WriteLookaround {
+        /// The index of the lookaround expression that matches
+        look_idx: SmallIndex,
+    },
```

Comment:
Sorry for being so nit-picky about naming. The codebase seems to follow these conventions. 

```suggestion
    /// This is like a match state but for a look-around expression
    /// executing this state will write a `true` into the look-around oracle at
    /// index `look_idx`.
    WriteLookaround {
        /// The index of the look-around expression that matches.
        lookaround_idx: SmallIndex,
    },
```

Reply:
here, in fact, the documentation is no longer correct :)